ecc/ecc.c

```diff
@@ -310,8 +310,8 @@ static void fieldModO(const uint32_t *A, uint32_t *result, uint8_t length) {
  * @return 1 if all elements in @val are zero, 0 otherwise.
  */
 static int is_zero(const uint32_t* val, size_t count) {
-  int result = 0;
-  int idx;
+  unsigned result = 0;
+  unsigned idx;
```

Comment:
I would like to see `size_t` here for `result` and `idx` even if this is not really necessary due to the length of the input data.

Reply:
Ah, sorry, I did not realize that this has already been fixed in #81.